data/autoyast_sle15/create_hdd/create_hdd_sap.xml.ep

```diff
@@ -0,0 +1,300 @@
+    <languages/>
+  </language>
+  <ntp-client>
+    <ntp_policy>auto</ntp_policy>
+  </ntp-client>
+  <software>
+    <products config:type="list">
+      % if ($check_var->('SLE_PRODUCT','sles4sap')) {
+      <product>SLES_SAP</product>
+      % } else {
+      <product><%= uc $get_var->('SLE_PRODUCT') %></product>
+      % }
+    </products>
+    <packages config:type="list">
+      <!-- Workaround for bsc#1202234: [addon]-release packages are missing after autoyast installation. -->
+      % foreach (values %$addons) {
+      <package><%= lc($_->{name}) %>-release</package>
+      % }
+      <!-- end of workaround -->
+      % if ($check_var->('SLE_PRODUCT', 'sles4sap')) {
+      <package>sle-module-sap-applications-release</package>
+      % }
+    </packages>
+    <patterns config:type="list">
+      % for my $pattern (@$patterns) {
+      <pattern><%= $pattern %></pattern>
+      % }
+    </patterns>
+  </software>
+  <services-manager>
+    % if ($check_var->('DESKTOP', 'gnome')) {
+    <default_target>graphical</default_target>
+    % }
+    % if ($check_var->('DESKTOP', 'textmode')) {
+    <default_target>multi-user</default_target>
+    % }
+    <services>
+      <disable config:type="list"/>
+      <enable config:type="list">
+        <service>sshd</service>
+      </enable>
+    </services>
+  </services-manager>
+  <timezone>
+    <hwclock>UTC</hwclock>
+    <timezone>Europe/Berlin</timezone>
+  </timezone>
+  <users config:type="list">
+    % unless ($check_var->('ROOTONLY', '1')) {
+    <user config:type="map">
+      <authorized_keys config:type="list"/>
+      <encrypted config:type="boolean">true</encrypted>
+      % if ($check_var->('ARCH', 'ppc64le')) {
+      <fullname>Bernhard M. Wiedemann</fullname>
+      % }
+      % unless ($check_var->('ARCH', 'ppc64le')) {
+      <fullname>bernhard</fullname>
+      % }
+      <gid>100</gid>
+      <home>/home/bernhard</home>
+      <password_settings config:type="map">
+        <expire/>
+        <flag/>
+        <inact>-1</inact>
+        <max>99999</max>
+        <min>0</min>
+        <warn>7</warn>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>1000</uid>
+      <user_password>$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/</user_password>
+      <username>bernhard</username>
+    </user>
+    % }
+    <user config:type="map">
+      <authorized_keys config:type="list"/>
+      <encrypted config:type="boolean">true</encrypted>
+      <fullname>root</fullname>
+      <gid>0</gid>
+      <home>/root</home>
+      <password_settings config:type="map">
+        <expire/>
+        <flag/>
+        <inact/>
+        <max/>
+        <min/>
+        <warn/>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>0</uid>
+      <user_password>$6$gdDHoMtVLjs4CCzf$2tSvAdgvqrKo84pA59bEjZRh7IGMfv4u0Yl4hrRzPgFPWLd8RXWdn/boT7yM3K3BlTk57qyR0TZ/nMb9rlpzx1</user_password>
```

Comment:
Same here.

Reply:
this is the encrypted password, this make sure the password is not leaked.